samples/transactional-session/aspnetcore-webapi/SqlPersistenceTS_6/WebApplication/MyHandler.cs

```diff
@@ -19,8 +19,7 @@ public async Task Handle(MyMessage message, IMessageHandlerContext context)
     {
         log.Info("Message received at endpoint");
 
-        var entity = await dataContext.MyEntities.Where(e => e.Id == message.EntityId)
-           .FirstAsync(cancellationToken: context.CancellationToken);
+        var entity = await dataContext.MyEntities.Where(e => e.Id == message.EntityId).FirstAsync();
```

Comment:
```suggestion
        var entity = await dataContext.MyEntities.Where(e => e.Id == message.EntityId)
           .FirstAsync();
```

Not hung up on this. It would be just more "consistent" with the v7 version